dss/util/auth/auth0.py

```diff
@@ -98,6 +98,14 @@ def security_flow(self, **kwargs):
             err += f'{", ".join(self.valid_methods)}'
             raise DSSException(500, err)
 
+        # Allow any action by admins
```

Comment:
This check to see if a user is an admin might seem to belong in the Authorize class.

However, how admins are treated depends on the auth system being used, so this is an implementation detail that is best left to the corresponding sub-class to check.